slug.go

```diff
@@ -75,6 +75,8 @@ func MakeLang(s string, lang string) (slug string) {
 		slug = SubstituteRune(slug, svSub)
 	case "tr", "tur":
 		slug = SubstituteRune(slug, trSub)
+	case "kz", "kk", "kaz":
+		slug = SubstituteRune(slug, kkSub)
```

Comment:
Could you move it to have all of langs in alphabetic order?

Reply:
done